highfidelity/hifi
# Case - 20550 install Nitpick on mac

https://highfidelity.fogbugz.com/f/cases/20550/
# Description
Adds nitpick to the Mac installer.
Nitpick is part of the development and PR builds, and is not part of the production build.
# Testing
Windows
======
1.  Download PR.
2.  Verify that a folder named nitpick is located in the installation folder
3.  Enter that folder and verify that `nitpick.exe` exists
4.  Double click on that file and verify that nitpick runs.

Mac
===
1.  Download the PR
2.  Verify that the DMG file contains the `nitpick` app
3.  Verify that the app runs
